src/test/java/bio/terra/service/snapshotbuilder/SnapshotBuilderTestData.java

```diff
@@ -53,42 +52,26 @@ public class SnapshotBuilderTestData {
                       .id(1)
                       .name("Year of birth")
                       .kind(SnapshotBuilderProgramDataOption.KindEnum.RANGE)
-                      .min(1900)
-                      .max(2023),
+                      .tableName("person")
+                      .columnName("year_of_birth"),
                   new SnapshotBuilderProgramDataOption()
                       .id(2)
                       .name("Ethnicity")
                       .kind(SnapshotBuilderProgramDataOption.KindEnum.LIST)
-                      .values(
-                          List.of(
-                              new SnapshotBuilderListOption().name("Hispanic or Latino").id(20),
-                              new SnapshotBuilderListOption().name("Not Hispanic or Latino").id(21),
-                              new SnapshotBuilderListOption().name("No Matching Concept").id(0))),
+                      .tableName("person")
+                      .columnName("ethnicity"),
                   new SnapshotBuilderProgramDataOption()
                       .id(3)
                       .name("Gender identity")
                       .kind(SnapshotBuilderProgramDataOption.KindEnum.LIST)
-                      .values(
-                          List.of(
-                              new SnapshotBuilderListOption().name("FEMALE").id(22),
-                              new SnapshotBuilderListOption().name("MALE").id(23),
-                              new SnapshotBuilderListOption().name("NON BINARY").id(24),
-                              new SnapshotBuilderListOption().name("GENDERQUEER").id(25),
-                              new SnapshotBuilderListOption().name("TWO SPIRIT").id(26),
-                              new SnapshotBuilderListOption().name("AGENDER").id(27),
-                              new SnapshotBuilderListOption().name("No Matching Concept").id(0))),
+                      .tableName("person")
+                      .columnName("gender_identity"),
                   new SnapshotBuilderProgramDataOption()
                       .id(4)
                       .name("Race")
                       .kind(SnapshotBuilderProgramDataOption.KindEnum.LIST)
-                      .values(
-                          List.of(
-                              new SnapshotBuilderListOption()
-                                  .name("American Indian or Alaska Native")
-                                  .id(28),
-                              new SnapshotBuilderListOption().name("Asian").id(29),
-                              new SnapshotBuilderListOption().name("Black").id(30),
-                              new SnapshotBuilderListOption().name("White").id(31)))))
+                      .tableName("person")
+                      .columnName("race")))
```

Comment:
Now that this data (race, gender, ethnicity) isn't stubbed here due to the refactor, where is it going to come from? Is there a new location where this info is stubbed until it can be actually derived from the dataset?

Reply:
Right now it doesn't exist - when I do the UI work, I am going to create a stubbed endpoint for them. I think there's a discussion to have here, maybe we can extend the columnStatistics endpoint to include list information? But more likely that will need a new endpoint.

Reply:
Okay so does this PR need to wait for the UI PR and the stubbed endpoint PR so as not to break anything? Yes I need to learn more, so would love to discuss, but expanding the columnStatistics endpoint sounds like it makes sense to me

Reply:
Because its a breaking API change, it is going to break things unless we did a versioned endpoint in the middle. Because this isn't a live feature, I think that is fine, but good instinct to be careful about that.